### PR TITLE
compute: implement structural search output command

### PR DESCRIPTION
**Pull request by rvantonder**

### CODE REVIEW

File: internal/compute/output_command.go [hunk 1]
```diff
@@ -29,14 +30,26 @@ func substituteRegexp(content string, match *regexp.Regexp, replacePattern, sepa
...
 func output(ctx context.Context, fragment string, matchPattern MatchPattern, replacePattern string, separator string) (*Text, error) {
-	var newFragment string
+	var newContent string
+	var err error
 	switch match := matchPattern.(type) {
 	case *Regexp:
-		newFragment = substituteRegexp(fragment, match.Value, replacePattern, separator)
+		newContent = substituteRegexp(fragment, match.Value, replacePattern, separator)
 	case *Comby:
-		return nil, nil
```
Comment (rvantonder) [inline internal/compute/output_command.go:37]: Key change: this case is implemented.

**Review by novoselrok — APPROVED** [review]

File: internal/compute/output_command_test.go [hunk 2]
```diff
@@ -26,6 +28,20 @@ func Test_output(t *testing.T) {
...
+	// If we are not on CI skip the test if comby is not installed.
+	if os.Getenv("CI") == "" && !comby.Exists() {
+		t.Skip("comby is not installed on the PATH. Try running 'bash <(curl -sL get.comby.dev)'.")
+	}
+
+	autogold.Want(
+		"structural search output",
+		`train(regional, intercity)
+train(commuter, lightrail)`).
+		Equal(t, test("Im a train. train(intercity, regional). choo choo. train(lightrail, commuter)", &Output{
```
Comment (camdencheek) [inline internal/compute/output_command_test.go:41]: what programming language is this?

**Review by camdencheek — APPROVED** [review]

File: internal/compute/output_command_test.go (hunk 2 above)
Comment (rvantonder) [reply internal/compute/output_command_test.go:41]: Oh, it's called Trainwreck. Surprised you have you not heard of it?

Comment (camdencheek) [reply internal/compute/output_command_test.go:41]: is this that cool new language you're gonna rewrite the sourcegraph frontend in? If so, I'm chugga-chugga-all-in.

Comment (rvantonder) [reply internal/compute/output_command_test.go:41]: Not at this point, it's a good language to use when everything is on fire.

**Pull request merged**